assets/conf/google/Caddyfile

```diff
@@ -0,0 +1,48 @@
+# This Caddyfile contains a full example of using Google
+# Authentication and Caddy.
+
+# This is a simplified version of the "multihost" example -- but only has one
+# hostname.
+
+example.com {
+	route /auth* {
+		authp {
+			crypto key sign-verify secretpassword
+            # All hosts using this auth must be in cookie_domain.
+			cookie domain example.com
+            # optional settings so auth lasts longer
+            crypto default token lifetime 604800
+			cookie lifetime 604800
+            # backend google <client_id> <client_secret>
+			# put your client_id and client_secret from Google Cloud Console here:
+			backend google 1234567890-a1b2c3d4e5f6g7.apps.googleusercontent.com abcd-efg-123456
+            # Here you can add specific roles to individual
+            # user accounts:
+            transform user {
```

Comment:
There's mixed tabs and spaces throughout the Caddyfile examples. Please use tabs.